### ISSUE TITLE
Details has extra focusable element when closed

### ISSUE THREAD

**Issue by Hifoz** [digdir/designsystemet#4019]
### Version number

1.4.0

### Description of the bug

After upgrading fra version 1.1.4 to 1.4.0, `<Details>` looks to have a second, 0-height element that can recieve focus whenever it is closed.

### Steps To Reproduce

See: https://codesandbox.io/p/sandbox/xms2gd
1. Put focus in the app
2. Tab through the closed panels
3. You should see a 0-height element catching focus inbetween the Detail-summarys

### Additional Information

<img width="1336" height="551" alt="Image" src="https://github.com/user-attachments/assets/42f999f5-db94-4784-b86b-ce06e6c234f1" />

**Comment by Febakke** [comment]
When testing I found that this bug only appeared with Firefox. We will investigate this.

**Comment by oddvernes** [comment]
It only happens in firefox, not chrome or safari
It does not happen with a native details/summary (e.g https://codepen.io/argyleink/pen/MYYpVjJ)
If i console.log document.activeElement on tab, What appears to happen is focus moves from `u-summary` to `u-details` which is the collapsed line. However i think more concretely it is `<slot part="details-content" ...>`. 
I think an adjustment in the [UHTMLDetailsStyle](https://github.com/u-elements/u-elements/blob/b309e4bdc69a818b18160e0c5c49e30c7f382721/packages/u-details/u-details.ts#L21C14-L21C31) can fix this issue, so pinging @eirikbacker 🙂  

Edit: it does not happen in the [u-details examples](https://u-elements.github.io/u-elements/elements/u-details), so perhaps something with our css overrides 🤔

**Comment by Hifoz** [comment]
Did som more testing myself, and it seems like if removing `overflow-y: hidden` from`.ds-details::part(details-content)` causes it to no longer recieve focus

<img width="570" height="132" alt="Image" src="https://github.com/user-attachments/assets/687e02aa-d569-4710-bc8e-faa97f47794f" />

**Comment by eirikbacker** [comment]
Right! As you point out @Hifoz, the `overflow` CSS property generates a tabstop in Firefox. I think consumers should not need to know about this/handle it - u-details should be more robust. Therefore we've released `u-details@1.0.5` now where `tabstop="-1"` is added to content-slot when closed, fixing the issue 🙌 
Thanks for reporting @Hifoz ! 👏 ☺️

**Comment by oddvernes** [comment]
Oh wow, today I learned (a new firefox behavior)! 

I have updated the package so this fix will be in the next release 👍
